sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuWindowExec.scala

```diff
@@ -483,6 +483,13 @@ object GpuWindowExec {
     val isFuncOkay = func match {
       case _: GpuBatchedRunningWindowWithFixer => true
       case GpuAggregateExpression(_: GpuBatchedRunningWindowWithFixer, _, _, _ , _) => true
+      case GpuNthValue(_, offset, _) =>
```

Comment:
Should we make `GpuBatchedRunningWindowWithFixer` have method that can enable/disable it for different situations?  Like First vs Last?

Reply:
I have moved the `FIRST`-specific code out from `GpuWindowFunctionExec`.  So the logic that decides whether to participate as a running window is now in `GpuFirst` and `GpuNthValue`.

Would you still recommend individual chicken-switches for these functions?